src/ReactiveUI/Xaml/ViewModelViewHost.cs

```diff
@@ -101,21 +101,20 @@ public ViewModelViewHost()
                     }
 
                     var viewLocator = ViewLocator ?? ReactiveUI.ViewLocator.Current;
-                    var view = viewLocator.ResolveView(x.ViewModel, x.Contract) ?? viewLocator.ResolveView(x.ViewModel, null);
+                    var view = viewLocator.ResolveView(x.ViewModel, x.Contract) ?? viewLocator.ResolveView(x.ViewModel);
```

Comment:
Hey Marc (@Silv3rcircl3) good catch on the subscription not being disposed. Can you share with me why this line was changed though?